### PR TITLE
tampil kontak

### PR DESCRIPTION
**Pull request by prasetyoanzery**
ada kok

### CODE REVIEW

**Review by wisnukurniawan — COMMENTED** [review]
Why theres no description!!

File: siswa/home.php [hunk 1]
```diff
@@ -18,8 +25,8 @@
 			<div class="divider"></div>
 			<div class="info_kontak">
 				<p>Info kontak guru</p>
-				<p>Email : - </p>
-				<p>No Hp : - </p>
+				<p>Email : <?php echo $kontak['no_hp']; ?></p>
+				<p>No Hp : <?php echo $kontak['email']; ?></p>
```
Comment (wisnukurniawan) [inline siswa/home.php:29]: Are you sure this run properly?

Comment (prasetyoanzery) [reply siswa/home.php:29]: yes, loading from db sir

Comment (elharies39) [inline siswa/home.php:29]: ini komen

Comment (wisnukurniawan) [reply siswa/home.php:29]: Approve then!

**Review by wisnukurniawan — APPROVED** [review]

File: siswa/home.php (hunk 1 above)
Comment (prasetyoanzery) [reply siswa/home.php:29]: yooo

**Review by elharies39 — APPROVED** [review]

**Pull request merged**